src/user/views/UserDashboard.vue

```diff
@@ -183,6 +184,9 @@ export default defineComponent({
       });
     }
 
+    const ownerEvents = await eventClient.getEventsOfOwner(user.value!.uid);
```

Comment:
@gokcedilek This is the problematic call — when setup is first called, `user.value` is going to be null because we're still fetching it in the background. Try putting this in the `else` block of the `watchEffect` call, user should be defined there.